include/EASTL/heap.h

```diff
@@ -275,20 +358,42 @@ namespace eastl
 	/// to make and maintain the heap.
 	///
 	template <typename RandomAccessIterator, typename Compare>
+	inline void pop_heap(RandomAccessIterator first, RandomAccessIterator last, Compare compare, typename eastl::enable_if<eastl::is_move_constructible<typename eastl::iterator_traits<RandomAccessIterator>::value_type>::value && !eastl::is_copy_constructible<typename eastl::iterator_traits<RandomAccessIterator>::value_type>::value>::type* = 0)
+	{
+		typedef typename eastl::iterator_traits<RandomAccessIterator>::difference_type difference_type;
+		typedef typename eastl::iterator_traits<RandomAccessIterator>::value_type      value_type;
+
+		value_type tempBottom( eastl::move(*(last - 1)) );
+		*(last - 1) = eastl::move( *first );
```

Comment:
Fixed the formatting...

I think I must've painted myself into a corner with a previous version of this change... Tested this with just using is_move_constructible and it compiles and passes. Will fix.